system-addon/content-src/components/ConfirmDialog/ConfirmDialog.jsx

```diff
@@ -1,7 +1,7 @@
-const React = require("react");
-const {connect} = require("react-redux");
-const {FormattedMessage} = require("react-intl");
-const {actionTypes, actionCreators: ac} = require("common/Actions.jsm");
+import {actionCreators as ac, actionTypes} from "common/Actions.jsm";
```

Comment:
Mildly irrelevant, but I think this was the only place where we import `actionTypes` but don't assign it to an `at` alias. 🤷‍♀️

Reply:
Noted, but I didn't really feel like making the change was worth it in this patch